media/css/firefox/mobile-download-desktop.less

```diff
@@ -0,0 +1,211 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+@import "../sandstone/fira-fonts.less";
```

Comment:
Hm, since the widget includes Sandstone, I think we'll have to go that route. Hopefully the stakeholders will be okay with deviating from the design a little.